### PR TITLE
executor: make the create/start/end time in admin show ddl jobs TZ aware

### PR DESCRIPTION
**Pull request by mjonss**
### What problem does this PR solve?

Issue Number: ref #26642 

Problem Summary:
ADMIN SHOW DDL JOBS did not adjust the create_time, start_time and end_time according to sessions time_zone.

### Check List

Tests 

- [X] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
ADMIN SHOW DDL JOBS now shows the times in the session's time_zone
```

### CODE REVIEW

File: executor/builder.go [hunk 1]
```diff
@@ -422,10 +422,19 @@ func (b *executorBuilder) buildShowDDL(v *plannercore.ShowDDL) Executor {
 }
 
 func (b *executorBuilder) buildShowDDLJobs(v *plannercore.PhysicalShowDDLJobs) Executor {
+	loc := b.ctx.GetSessionVars().TimeZone
```
Comment (wjhuang2016) [inline executor/builder.go:425]: Using getTimeZone() is enough?

Comment (mjonss) [reply executor/builder.go:425]: Yes, if it were exported :)

Do you want me to move the `getTimeZone()` function to `util/timeutil/time.go` and export it from there instead?

Comment (mjonss) [reply executor/builder.go]: I used the SessionVars `Location()` function instead, which is similar.

**Review by wjhuang2016 — APPROVED** [review]

**Review by djshow832 — APPROVED** [review]

**Pull request merged**